components/infobox/wikis/rocketleague/infobox_person_player.lua

```diff
@@ -0,0 +1,253 @@
+---
+-- @Liquipedia
+-- wiki=rocketleague
+-- page=Module:Infobox/Person/Player
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Player = require('Module:Infobox/Person')
+local String = require('Module:StringUtils')
+local Class = require('Module:Class')
+local Earnings = require('Module:Earnings')
+local Namespace = require('Module:Namespace')
+local Variables = require('Module:Variables')
+local Page = require('Module:Page')
+local YearsActive = require('Module:YearsActive')
+local Matches = require('Module:Matches_Player')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+
+local _BANNED = mw.loadData('Module:Banned')
+
+local _pagename = mw.title.getCurrentTitle().prefixedText
+
+local CustomPlayer = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+local _START_YEAR = '2015'
+local _CURRENT_YEAR = mw.getContentLanguage():formatDate('Y')
+
+function CustomPlayer.run(frame)
+	local player = Player(frame)
+	_args = player.args
+	player.args.informationType = player.args.informationType or 'Player'
+
+	player.calculateEarnings = CustomPlayer.calculateEarnings
+	player.adjustLPDB = CustomPlayer.adjustLPDB
+	player.defineCustomPageVariables = CustomPlayer.defineCustomPageVariables
+	player.createBottomContent = CustomPlayer.createBottomContent
+	player.getCategories = CustomPlayer.getCategories
+	player.createWidgetInjector = CustomPlayer.createWidgetInjector
+
+	return player:createInfobox(frame)
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'status' then
+		local statusContents = {}
+		local status
+		if not String.isEmpty(_args.status) then
+			status = Page.makeInternalLink({onlyIfExists = true}, _args.status) or _args.status
+		end
+		local banned = _BANNED[string.lower(_args.banned or '')]
+		if not banned and not String.isEmpty(_args.banned) then
+			banned = '[[Banned Players/Other|Multiple Bans]]'
+		end
+		local banned2 = _BANNED[string.lower(_args.banned2 or '')]
+		local banned3 = _BANNED[string.lower(_args.banned3 or '')]
+		table.insert(statusContents, status)
+		table.insert(statusContents, banned)
+		table.insert(statusContents, banned2)
+		table.insert(statusContents, banned3)
+
+		local yearsActive = _args.years_active
+		if String.isEmpty(yearsActive) then
+			yearsActive = YearsActive.get({player=mw.title.getCurrentTitle().baseText})
```

Comment:
Do we not have the pagename in our instance?

Reply:
it is not pagename it is `{{BASEPAGENAME}}`

![Screenshot 2021-11-09 12 17 23](https://user-images.githubusercontent.com/75081997/140914712-67f659da-21f5-461c-8261-31e1ad7e462c.png)